### PR TITLE
new folder for promo screen and alt warning fixes

### PR DESCRIPTION
**Pull request by lloydweston**

### CODE REVIEW

File: src/app/promoScreen/PromoScreen.js [hunk 1]
```diff
@@ -54,7 +53,7 @@ const PromoScreen = () => (
             display: 'flex', justifyContent: 'center', alignItems: 'center', height: '100%', flexDirection: 'column',
           }}
           >
-            <img src={colobus} style={{ marginBottom: 40 }} />
+            <img src={colobus} alt='' style={{ marginBottom: 40 }} />
```
Comment (lauri3new) [inline src/app/promoScreen/PromoScreen.js]: https://blog.hubspot.com/marketing/image-alt-text 
What word best represents this image?

File: src/App.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { useAuthState } from 'react-firebase-hooks/auth'
 import firebase from './capabilities/firebase'
 import './App.css'
-import PromoScreen from './app/PromoScreen'
+import PromoScreen from './app/promoScreen/PromoScreen'
```
Comment (lauri3new) [inline src/App.js:5]: is promo screen the best name? I think so

File: src/app/promoScreen/PromoScreen.js (hunk 1 above)
Comment (lloydweston) [reply src/app/promoScreen/PromoScreen.js]: updated :)

File: src/app/promoScreen/PromoScreen.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
 import React from 'react'
-import { Button } from 'antd'
-import FeatureCol from '../common/FeatureCol'
```
Comment (lauri3new) [inline src/app/promoScreen/PromoScreen.js:3]: haha good one

**Review by lauri3new — CHANGES_REQUESTED** [review]

File: src/app/promoScreen/PromoScreen.js (hunk 1 above)
Comment (lauri3new) [reply src/app/promoScreen/PromoScreen.js]: Very nice

File: src/App.js (hunk 2 above)
Comment (lauri3new) [reply src/App.js:5]: on second thoughts this is fine

**Review by lauri3new — APPROVED** [review]

**Pull request merged**